src/test/java/org/sonatype/gradle/plugins/scan/ossindex/OssIndexAuditTaskTest.java

```diff
@@ -99,6 +99,7 @@ private OssIndexAuditTask buildAuditTaskSpy(boolean isSimulated) {
 
     OssIndexPluginExtension extension = new OssIndexPluginExtension(project);
     extension.setSimulationEnabled(isSimulated);
+    extension.setDependencyGraph(true);
```

Comment:
Why was this necessary?
Is the message `Vulnerabilities detected, check log output to review them` gone in the new output?

Reply:
> Is the message `Vulnerabilities detected, check log output to review them` gone in the new output?

No, we still included that message.

> Why was this necessary?

I should of looked at the error message closer. For the default/table report, a `NullPointerException` was being thrown because we don't set the vulnerability description in the test setup (description isn't used in the dependency graph report). I've add a default value in case of null (shouldn't happen though ;-)). See b1836685